doc/RecordingControl.xml

```diff
@@ -1470,6 +1470,62 @@ Change Request 2061, 2063, 2065, 2109</revremark>
       </variablelist>
       <para>The requested operation does not exist.</para>
     </section>
+    <section>
+      <title>OverrideSegmentDuration</title>
+      <para>OverrideSegmentDuration dynamically updates a RecordingConfiguration to produce smaller
+        recording segments for a limited time. This allows for a client to request faster access to recorded data while
+        reducing the overall storage costs during normal operation.
+      </para>
+      <para>
+        When the temporary override is active, the device should attempt to close its currently recording segment and then
+        shall record further segments with the new duration provided in this command. The override shall be active for the
+        duration specified in the command, once the command expires the device shall resume recording with the original segment
+        duration on its next segment.
+
+        When a new override request is sent before the previous override has expired, the new override shall replace the 
+        previous override and the new expiration shall apply.
+      </para>
+      <para>
+        The active override value, if any exists, shall be shown in the corresponding <literal>RecordingConfiguration</literal>.
+      </para>
+      <para>
+        A device that indicates support for OverrideSegmentDuration shall support this command.
+      </para>
+      <variablelist role="op">
+        <varlistentry>
+          <term>request</term>
+          <listitem>
+            <para role="param">TargetSegmentDuration [xs:duration]</para>
```

Comment:
Shouldn't client query segment duration limits (expressed as options?) for client to check what device can accept and allow device to throw a relevant error message if the duration is too small or high?

Reply:
I agree with this comment and I feel further clarity is needed.

Reply:
I'm unsure on that point. This parameter is simply the same as the TargetSegmentDuration of the RecordingConfiguration Target setting. It has the same limits, which is none. I'm not sure why there were no limits documented for the RecordingConfiguration, when Cloud Recording was done, as I was not present then.

Do you know why such limits were not documented or specified at that moment? I'm assuming here that this was discussed, and was voluntarily left open ended. Is it not the case?

Reply:
I personally don't remember if it was discussed or not. But if we want to specify a limit now, I suggest to have it done in its own PR since it is unrelated to this new request.

Reply:
@jmelancongen @jflevesque-genetec I agree.